Assets/Scripts/KuriTextManager.cs

```diff
@@ -5,6 +5,9 @@
 
 namespace MoveToCode {
     public class KuriTextManager : Singleton<KuriTextManager> {
+        //public GameObject KuriLight0;
+        public GameObject[] lights;
```

Comment:
It looks like for the lights you are only using the "MeshRenderer" so you can probably just hold an array of "MeshRender"s instead of the "GameObjects"; You should be able to also fill this array programmatically by finding the lights (they are probably a list of children) through something like  `GetComponentsInChildren<MeshRenderer>()` : https://docs.unity3d.com/ScriptReference/Component.GetComponentsInChildren.html